packages/asset-swapper/src/utils/market_operation_utils/sampler_operations.ts

```diff
@@ -84,15 +84,13 @@ export class SamplerOperations {
         public readonly provider?: SupportedProvider,
         public readonly balancerPoolsCache: BalancerPoolsCache = new BalancerPoolsCache(),
         public readonly creamPoolsCache: CreamPoolsCache = new CreamPoolsCache(),
-        protected readonly getBancorServiceFn?: () => BancorService, // for dependency injection in tests
         protected readonly tokenAdjacencyGraph: TokenAdjacencyGraph = { default: [] },
         public readonly liquidityProviderRegistry: LiquidityProviderRegistry = LIQUIDITY_PROVIDER_REGISTRY,
-    ) {}
+    ) {
+        void this.getBancorServiceAsync();
```

Comment:
This syntax is new to me. Does it not work without `void`?
This is to prime the pump, right? Can we drop a comment mentioning that?

Reply:
It's a hack to avoid lint errors as we don't want to await it and we don't care about what it returns. 

I've reworked to 

```
        this.initBancorServiceAsync().catch(/* do nothing */);
```